asaps/models.py

```diff
@@ -184,33 +180,39 @@ def extract_fields(client, repo_id, rec_type, field):
                             'relatedmaterial', 'acqinfo', 'arrangement',
                             'processinfo', 'bibliography']
         if field in note_type_fields:
-            report_dict = extract_note_field(field, rec_obj, report_dict)
+            report_dicts = extract_note_field(field, rec_obj, report_dict,
+                                              logger)
+            for report_dict in report_dicts:
```

Comment:
I'm curious what's going on here? You are discarding everything except for the last value of the `report_dicts` generator. If that is indeed the intended behavior, this would indicate a deeper problem we should dig into.